### PR TITLE
Makes wikibooks not entirely shit

### PR DESCRIPTION
**Pull request by alexkar598**
# General Documentation

### Intent of your Pull Request

Makes wikibooks not shit by using the mediawiki api instead of using deprecated shit.

The rest of the code is at https://wiki.yogstation.net/frame.html . It has to be there because BYOND is an insecure context and internet explorer will refuse to let BYOND speak to our HTTPS wiki. STONKS.

### Why is this change good for the game?

Because wiki books sucks

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Rn wikibooks = cringe and never used
After this pr, wikibooks = never used

### What should players be aware of when it comes to the changes your PR is implementing?
That wikibooks are slightly less shit

### What general grouping does this PR fall under? 
Gaming

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
Gaming

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
Gaming x2

# Changelog

:cl:  
tweak: Wikibooks have been improved to suck less
/:cl:

# Note to wiki men
This brings two new css classes to the wiki
game-only : Will not be displayed on the wiki, only in game books
wiki-only: Will be displayed on the wiki, not in in game books, currently used to hide shit in space law so it doesnt look awful in game.

By default, elements are displayed in both the wiki and in the in game books

### CODE REVIEW

**Comment by Anvilman6** [comment]
Excellent and alexkar-pilled

**Comment by alexkar598** [comment]
Old one (bad and cringe): 
New one (good and nice):

**Comment by optimumtact** [comment]
put the contents of the frame in a file in the repo too

**Comment by maxion12345** [comment]
make em printable in curator console next

**Review by JamieD1 — APPROVED** [review]
